Documentation/DebugMenu.md

```diff
@@ -13,15 +13,15 @@ Items are shown in the menu on a single line, so they use short codes and appear
 
 ### ID
 - This is used by Irons that have an ID and serial number to help check if the iron is authentic. All Pinecil V1 show the same ID number as this is the number programmed into the MCU.
-- The new Pinecil V2 released Aug. 2, 2022 now uses BL706, which enables generating a unique ID/Serial number to every iron. This can be used to verify your Pinecil authenticity [here](https://pinecil.pine64.org/).
+- The new Pinecil V2 released Aug. 2, 2022 now uses MCU BL706, which enables generating a unique ID/Serial number to every iron. This can be used to verify your [Pinecil authenticity here](https://pinecil.pine64.org/).
 
-### UpTime
+### Time
 
-This just shows how many deciseconds the unit has been powered for.
+This is just uptime, and shows how many deciseconds the unit has been powered for.

```

Comment:
```suggestion
This just shows how many deciseconds the unit has been powered for.
```
Please revert this as well.
The way you put it doesn't make it any better. The comma is not even needed there.